src/core/Akka/Util/Internal/Collections/ListExtensions.cs

```diff
@@ -0,0 +1,31 @@
+//-----------------------------------------------------------------------
+// <copyright file="ListExtensions.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2017 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+
+namespace Akka.Util.Internal.Collections
+{
+    public static class ListExtensions
+    {
+        public static List<T> Shuffle<T>(this List<T> @this)
+        {
+            var list = new List<T>(@this);
+            var rng = new Random();
```

Comment:
thank you but I thought to replace with Math.round and Math.random as Java implementation